packages/solid/utils/handleNavigation.ts

```diff
@@ -76,7 +77,7 @@ export function handleNavigation(direction: 'up' | 'right' | 'down' | 'left'): K
     assertTruthy(active instanceof ElementNode);
     this.onSelectedChanged && this.onSelectedChanged.call(this, this, active, this.selected, lastSelected);
 
-    if (this.plinko && lastSelected !== undefined) {
+    if (this.plinko) {
```

Comment:
why are we removing this check?

Reply:
const lastSelected = this.selected || 0;

can never be undefined